<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Relatorio</class>
 <widget class="QWidget" name="Relatorio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>636</width>
    <height>695</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Relat√≥rio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../parkmanager.qrc">
    <normaloff>:/icons/documentation.png</normaloff>:/icons/documentation.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QDateEdit" name="datainiDE">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="locale">
        <locale language="Portuguese" country="Brazil"/>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="datainiLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Data Inicial:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="datafinDE">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="locale">
        <locale language="Portuguese" country="Brazil"/>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QPushButton" name="procurarButton">
       <property name="text">
        <string>Pr&amp;ocurar</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/find.png</normaloff>:/icons/find.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="datafinLabel">
       <property name="text">
        <string>Data Final:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resTable">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="csvButton">
       <property name="text">
        <string>Gerar CS&amp;V</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/documentation.png</normaloff>:/icons/documentation.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fecharButton">
       <property name="text">
        <string>&amp;Fechar</string>
       </property>
       <property name="icon">
        <iconset resource="../parkmanager.qrc">
         <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>datainiDE</tabstop>
  <tabstop>datafinDE</tabstop>
  <tabstop>procurarButton</tabstop>
  <tabstop>resTable</tabstop>
  <tabstop>csvButton</tabstop>
  <tabstop>fecharButton</tabstop>
 </tabstops>
 <resources>
  <include location="../parkmanager.qrc"/>
 </resources>
 <connections/>
</ui>
